### PR TITLE
fix(driving-instructor-registrations): adding feature flags

### PR DESCRIPTION
**Pull request by albinagu**
## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

**Review by alexdiljar — APPROVED** [review]
Brilliant!

File: libs/feature-flags/src/lib/features.ts [hunk 1]
```diff
@@ -15,4 +15,6 @@ export enum Features {
   // Application visibility flags
   exampleApplication = 'isExampleApplicationEnabled',
   accidentNotification = 'isAccidentNotificationEnabled',
+  drivingInstructorRegistrations = 'isDrivingInstructorRegistrationsEnabled',
+  drivingSchoolConfirmation = 'isDrivingSchoolConfirmationsEnabled',
```
Comment (saevarma) [inline libs/feature-flags/src/lib/features.ts:19]: This is a complete nitpick: Either add the plural 's' to the property name, or remove it from the configcat key
```suggestion
  drivingSchoolConfirmations = 'isDrivingSchoolConfirmationsEnabled',
```
or
```suggestion
  drivingSchoolConfirmation = 'isDrivingSchoolConfirmationEnabled',
```

**Review by saevarma — APPROVED** [review]
Nice!

File: libs/feature-flags/src/lib/features.ts (hunk 1 above)
Comment (albinagu) [reply libs/feature-flags/src/lib/features.ts:19]: will do! 😄

**Pull request merged**